src/PKSim.CLI.Core/Services/ExportSimulationRunner.cs

```diff
@@ -64,30 +73,43 @@ private async Task exportProject(ExportRunOptions runOptions)
          _logger.AddInfo($"Project export for '{projectFile}' terminated");
       }
 
-      public async Task ExportSimulationsIn(PKSimProject project, ExportRunOptions exportRunOptions)
+      public async Task<SimulationExport[]> ExportSimulationsIn(PKSimProject project, ExportRunOptions exportRunOptions)
       {
          var nameOfSimulationsToExport = (exportRunOptions.Simulations ?? Enumerable.Empty<string>()).ToList();
          if (!nameOfSimulationsToExport.Any())
             nameOfSimulationsToExport.AddRange(project.All<Simulation>().AllNames());
 
+         var simulationExports = new List<SimulationExport>();
+
          //sequential for now
          foreach (var simulationName in nameOfSimulationsToExport)
          {
             var simulation = project.BuildingBlockByName<Simulation>(simulationName);
             if (simulation == null)
             {
-               _logger.AddWarning($"Simulation '{simulationName}' was not found in project '{project.Name}'");
+               _logger.AddWarning($"Simulation '{simulationName}' was not found in project '{project.Name}'", project.Name);
```

Comment:
Do not understand this change. Why is `, project.Name` at the end of the call required?

Reply:
This is a feature that we did not use so far but I had implemented it anyways just in case: It's a category for log. If you don't specify anything, it uses the name of the app (PK-Sim). But if you do specify something, it uses that instead. 

Output
```
dbug: PK-Sim[0]
      Starting qualification run
dbug: PK-Sim[0]
      Reading configuration from file 'C:\tests\7.5\Inputs\config.json'
dbug: Itraconazole[0]
      Loading project 'Itraconazole' from snapshot...
dbug: Itraconazole[0]
      Loading simulation 'S1'...
dbug: Itraconazole[0]
      Exporting project Itraconazole to 'C:\tests\7.5\Outputs\Itraconazole'
dbug: Itraconazole[0]
      Exporting simulation results to 'C:\tests\7.5\Outputs\Itraconazole\S1\S1.csv'
dbug: Itraconazole[0]
      Exporting simulation parameters to 'C:\tests\7.5\Outputs\Itraconazole\S1\S1_parameters.csv'
dbug: Itraconazole[0]
      Exporting simulation results to 'C:\tests\7.5\Outputs\Itraconazole\S1\S1.json'
dbug: Itraconazole[0]
      Exporting simulation SimModel xml to 'C:\tests\7.5\Outputs\Itraconazole\S1\S1.xml'
dbug: Itraconazole[0]
      Exporting simulation pkml to 'C:\tests\7.5\Outputs\Itraconazole\S1\S1.pkml'
dbug: Itraconazole[0]
      Simulation 'S1' exported to 'C:\tests\7.5\Outputs\Itraconazole\S1'
dbug: Itraconazole[0]
      Observed data 'Itraconazole 600mg MD' exported to 'C:\tests\7.5\Outputs\ObservedData\Itraconazole 600mg MD.xlsx'
```

As you can see, instead of PKSim, we now now what the entry relates to (e.g. Itraconazole). Redundant when we only have ONE project, but very useful once we start to have multiple projects ast the same time. You can then group entries by category for example